cmd/zdb/zdb.c

```diff
@@ -1404,14 +1410,19 @@ blkid2offset(const dnode_phys_t *dnp, const blkptr_t *bp,
 }
 
 static void
-snprintf_blkptr_compact(char *blkbuf, size_t buflen, const blkptr_t *bp)
+snprintf_blkptr_compact(char *blkbuf, size_t buflen, const blkptr_t *bp,
+    boolean_t free)
```

Comment:
Similarly, the name "free" is a bit problematic. show_free would be more descriptive, but a shorter version of the name would be fine.